scripts/fix_user_mailchimp.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import sys
+from datetime import datetime
+
+from django.db import transaction
+from django.utils import timezone
+
+from website.app import setup_django
+setup_django()
+from osf.models import OSFUser
+from scripts import utils as script_utils
+from website.mailchimp_utils import subscribe_mailchimp
+from website import settings
+
+logger = logging.getLogger(__name__)
+
+
+def main():
+    dry = '--dry' in sys.argv
+    if not dry:
+        # If we're not running in dry mode log everything to a file
+        script_utils.add_file_logger(logger, __file__)
+
+    with transaction.atomic():
+        start_time = datetime.strptime('2017-12-20 08:25:25', '%Y-%m-%d %H:%M:%S')
```

Comment:
Minor: It's preferable to build a datetime directly,

```
import pytz
from datetime import datetime

start_time = datetime(2017, 12, 20, 8, 25, 25, tzinfo=pytz.UTC)
```

Reply:
fixed in 5b11b1f